maintainers/scripts/remove-old-aliases.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i python3 -p "python3.withPackages(ps: with ps; [ dateutil ])"
+# example usage:
+# $ NIX_PATH=nixpkgs=$PWD ./maintainers/scripts/remove-old-aliases.py --year 2018 --file ./pkgs/top-level/aliases.nix --dry-run
+# Also check this file with mypy after every change!
```

Comment:
I removed all globals and added some docs.

Reply:
thanks